bundle/src/main/java/com/adobe/acs/commons/widgets/MultiFieldPanelFunctions.java

```diff
@@ -57,7 +57,7 @@ private MultiFieldPanelFunctions() {
     public static List<Map<String, String>> getMultiFieldPanelValues(Resource resource, String name) {
         ValueMap map = resource.adaptTo(ValueMap.class);
```

Comment:
just for future reference (no reason to change this now), since we're now requiring AEM 6.0, a slightly better solution would be to replace this line with

    ValueMap map = resource.getValueMap();

That will guarantee that map is a non-null object.